code/modules/research/designs/bluespace_designs.dm

```diff
@@ -13,6 +13,16 @@
 	category = list("Bluespace Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE | DEPARTMENTAL_FLAG_ENGINEERING | DEPARTMENTAL_FLAG_CARGO | DEPARTMENTAL_FLAG_SECURITY
 
+/datum/design/dragnetbeacon
+	name = "DROPnet"
+	desc = "A beacon marking the location where criminals will be teleported via DRAGnet devices."
+	id = "dragnetbeacon"
+	build_type = PROTOLATHE
+	materials = list(/datum/material/iron = 150, /datum/material/glass = 200)
```

Comment:
That is cheap as fuck for something bluespace

Reply:
Regular beacon is 150/150